### PR TITLE
rbd-mirror: rename on_stop_journal_replay() and some operator<< arguments

### PR DESCRIPTION
**Pull request by nbalacha**
Minor code cleanup changes.

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "quincy"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

- When filling out the below checklist, you may click boxes directly in the GitHub web UI.  When entering or editing the entire PR message in the GitHub web UI editor, you may also select a checklist item by adding an `x` between the brackets: `[x]`.  Spaces and capitalization matter when checking off items this way.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [x] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [x] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
- `jenkins test rook e2e`
</details>

### CODE REVIEW

File: src/tools/rbd_mirror/Types.cc [hunk 1]
```diff
@@ -11,19 +11,19 @@ std::ostream &operator<<(std::ostream &os, const ImageId &image_id) {
             << "id=" << image_id.id;
 }
 
-std::ostream& operator<<(std::ostream& lhs,
-                         const LocalPoolMeta& rhs) {
-  return lhs << "mirror_uuid=" << rhs.mirror_uuid;
+std::ostream& operator<<(std::ostream& os,
+                         const LocalPoolMeta& lpm) {
```
Comment (idryomov) [inline src/tools/rbd_mirror/Types.cc:15]: It would be nicer to match `local_pool_meta` and `remote_pool_meta` names used in the declarations.

Comment (nbalacha) [reply src/tools/rbd_mirror/Types.cc]: Done.

File: src/tools/rbd_mirror/ImageReplayer.h [hunk 2]
```diff
@@ -142,7 +142,7 @@ class ImageReplayer {
   bool on_start_interrupted();
```
Comment (idryomov) [inline src/tools/rbd_mirror/ImageReplayer.h:142]: Let's also change `JOURNAL_REPLAY_SHUT_DOWN` to `REPLAY_SHUT_DOWN` in the ASCII art a few lines.

Comment (nbalacha) [reply src/tools/rbd_mirror/ImageReplayer.h:142]: Done.

**Comment by idryomov** [comment]
jenkins test make check

**Comment by idryomov** [comment]
jenkins test windows

**Comment by idryomov** [comment]
jenkins test windows

**Comment by idryomov** [comment]
jenkins test make check

**Comment by idryomov** [comment]
jenkins test windows

**Review by idryomov — APPROVED** [review]

**Pull request merged**